front/src/modules/rollingStock/components/RollingStockEditor/RollingStockEditorFormHelpers.tsx

```diff
@@ -335,78 +319,58 @@ export const RollingStockEditorParameterForm = ({
 
 type RollingStockEditorOnboardSystemEquipmentFormProps = {
   rollingStockValues: RollingStockParametersValues;
-  setRollingStockValues: (
-    rollingStockValue: React.SetStateAction<RollingStockParametersValues>
-  ) => void;
-  rollingStockSchemasProperties: readonly SchemaProperty[];
+  setRollingStockValues: (rollingStockValue: RollingStockParametersValues) => void;
 };
 
-const RollingStockEditorOnboardSystemEquipmentFormColumn = ({
+export const RollingStockEditorOnboardSystemEquipmentForm = ({
   rollingStockValues,
   setRollingStockValues,
-  rollingStockSchemasProperties,
-  refList,
-}: RollingStockEditorOnboardSystemEquipmentFormProps & {
-  refList: string[];
-}) => {
+}: RollingStockEditorOnboardSystemEquipmentFormProps) => {
   const { t } = useTranslation(['rollingstock']);
 
+  const rollingStockSchemasPropertiees = useCompleteRollingStockSchemasProperties();
+
+  const rsSigSystemList = rollingStockValues.supportedSignalingSystems;
+  const sigSystemProperty = rollingStockSchemasPropertiees.filter(
+    (property) => property.title === 'supportedSignalingSystems'
+  )[0];
+
+  const updateSigSystemsList = (value: string) => (e: React.ChangeEvent<HTMLInputElement>) => {
+    const newList = e.target.checked
+      ? [...rsSigSystemList, value]
+      : rsSigSystemList.filter((v) => v !== value);
+    setRollingStockValues({
+      ...rollingStockValues,
+      supportedSignalingSystems: newList,
+    });
```

Comment:
You can remove rollingStockValues from the props and only give rsSignalingSystems (=rollingStockValues.supportedSignalingSystems) as a prop
```suggestion
    setRollingStockValues((prevRollingStockValues) => {
      ...prevRollingStockValues,
      supportedSignalingSystems: newList,
    });
```